code/game/machinery/big_manipulator.dm

```diff
@@ -234,6 +390,64 @@
 /obj/machinery/big_manipulator/proc/finish_rotate_animation(backward)
 	animate(manipulator_hand, transform = matrix(180 * backward, MATRIX_ROTATE), working_speed*0.5)
 
+/obj/machinery/big_manipulator/proc/check_filter(obj/item/what_item)
+	var/filtered_obj = filter_obj?.resolve()
+	if(!istype(what_item, selected_type))
+		return
+	/// We use filter only on items. closets, humans and etc don't need filter check.
+	if(istype(what_item, /obj/item))
+		if((filtered_obj && !istype(what_item, filtered_obj)))
+			return FALSE
+	return TRUE
+
+/// Create dummy to force him use our item and then delete him.
+/obj/machinery/big_manipulator/proc/create_abstract_living()
+	var/mob/living/carbon/human/abstract_living = new /mob/living/carbon/human(get_turf(src))
+	abstract_living.icon_state = "blank"
```

Comment:
```suggestion
```
This won't do anything for humans, since they're actually composed of dozens of meticolously layered overlays.